lmfit/models.py

```diff
@@ -983,6 +984,57 @@ def guess(self, data, x=None, negative=False, **kwargs):
     guess.__doc__ = COMMON_GUESS_DOC
 
 
+class ThermalDistributionModel(Model):
+    r"""Return a thermal distribution function. Variable ``kind`` defines
+    the kind of distribution as below with parameters ``amplitude``
+    (:math:`A`), ``center`` (:math:`x_0`) and ``kT`` (:math:`kT`).
+
+    .. math::
+
+        f(x; A, x_0, kT, T) = \frac{1}{A \exp(\frac{x - x_0}{kT}) + kind}
+
+    As such, these thermal distributions are available:
+
+    - Bose-Einstein distribution via :math:`kind=-1` (the default)
+    - Maxwell-Boltzmann distribution via :math:`kind=0`
+    - Fermi-Dirac distribution via :math:`kind=1`
+
+    see http://hyperphysics.phy-astr.gsu.edu/hbase/quantum/disfcn.html
+
+    Comments:
+
+    - Amplitude and kind are set not to vary by default.
+    - kT should be defined in the same units as x (kB = 8.617e-5 eV/K).
+    - set :math:`kT<0` to implement the energy loss convention common in
+      scattering research.
+    """
+    def __init__(self, independent_vars=['x'], prefix='', nan_policy='raise',
+                 **kwargs):
+        kwargs.update({'prefix': prefix, 'nan_policy': nan_policy,
+                       'independent_vars': independent_vars})
+        super().__init__(thermal_distribution, **kwargs)
+        self._set_paramhints_prefix()
+
+    def _set_paramhints_prefix(self):
+        self.set_param_hint('amplitude', min=0, max=1, vary=False)
```

Comment:
sorry if this should be obvious... but is there a reason why ```amplitude``` can only be between 0 and 1? If there isn't one should probably not set a ```max``` here.

Reply:
The solutions with amplitude > 1 would be non-physical in the usual use of the equation, but this could be left free. I'll remove this nonetheless.

Reply:
I have no strong preference and happily leave the decision to you.

Reply:
I would suggest not using any of these hints.  That is, we should sort of prefer to use put such hints here for things that are "mathematical infeasible" versus "physically non-sensical".

Reply:
it's removed